packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.service.js

```diff
@@ -0,0 +1,63 @@
+import compact from 'lodash/compact';
+import map from 'lodash/map';
+import 'moment';
+
+export default class PciStoragesObjectStorageService {
+  /* @ngInject */
+  constructor($http, $q, OvhApiCloudProjectUser) {
+    this.$http = $http;
+    this.$q = $q;
+    this.OvhApiCloudProjectUser = OvhApiCloudProjectUser;
+  }
+
+  getUserDetails(projectId, userId) {
+    return this.$http
+      .get(`/cloud/project/${projectId}/user/${userId}`)
+      .then(({ data }) => data);
+  }
+
+  removeAllCredentials(projectId, userId, credentials) {
+    return this.$q.all(
+      map(credentials, (credential) =>
+        this.$http.delete(
+          `/cloud/project/${projectId}/user/${userId}/s3Credentials/${credential.access}`,
+        ),
+      ),
+    );
+  }
```

Comment:
Not sure how this behaves if user has lots of credentials, can't a bulk be provided by API ?

Reply:
There is no bulk delete API and i checked with PU, users will not have lot of credentials so this should be ok.